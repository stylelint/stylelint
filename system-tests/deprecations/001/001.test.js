/* @flow */
"use strict"

const systemTestUtils = require("../../systemTestUtils")
const stylelint = require("../../../lib")

it("deprecations/001", () => {
  return stylelint.lint({
    files: [systemTestUtils.caseStylesheetGlob("deprecations/001")],
    configFile: systemTestUtils.caseConfig("deprecations/001"),
  }).then((output) => {
    expect(systemTestUtils.prepResults(output.results)).toMatchSnapshot()
  })
})
