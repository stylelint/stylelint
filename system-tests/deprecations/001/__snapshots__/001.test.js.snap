// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deprecations/001 1`] = `
Array [
  Object {
    "deprecations": Array [
      Object {
        "reference": "https://stylelint.io/user-guide/rules/block-no-single-line/",
        "text": "'block-no-single-line' has been deprecated and in 8.0 will be removed. Instead use 'block-opening-brace-newline-after' and 'block-closing-brace-newline-before' with the \\"always\\" option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/declaration-block-no-ignored-properties/",
        "text": "'declaration-block-no-ignored-properties' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/declaration-block-properties-order/",
        "text": "'declaration-block-properties-order'has been deprecated and in 8.0 will be removed. Instead use the community 'stylelint-order' plugin pack.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/media-feature-no-missing-punctuation/",
        "text": "'media-feature-no-missing-punctuation' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/selector-no-empty/",
        "text": "'selector-no-empty' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/no-browser-hacks/",
        "text": "'no-browser-hacks' has been deprecated and in 8.0 will be removed. Use 'stylelint-no-browser-hacks' plugin instead.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/no-indistinguishable-colors/",
        "text": "'no-indistinguishable-colors' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/no-unsupported-browser-features/",
        "text": "'no-unsupported-browser-features' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/stylelint-disable-reason/",
        "text": "'stylelint-disable-reason' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/custom-property-no-outside-root/",
        "text": "'custom-property-no-outside-root' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/root-no-standard-properties/",
        "text": "'root-no-standard-properties' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/rule-nested-empty-line-before/",
        "text": "'rule-nested-empty-line-before' has been deprecated and in 8.0 will be removed. Instead use 'rule-empty-line-before'.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/rule-non-nested-empty-line-before/",
        "text": "'rule-non-nested-empty-line-before' has been deprecated and in 8.0 will be removed. Instead use 'rule-empty-line-before'.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/selector-root-no-composition/",
        "text": "'selector-root-no-composition' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/time-no-imperceptible/",
        "text": "'time-no-imperceptible' has been deprecated and in 8.0 will be removed. Instead use 'time-min-milliseconds' with '100' as its primary option.",
      },
    ],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "source": "deprecations/001/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 3,
        "line": 1,
        "rule": "block-no-single-line",
        "severity": "error",
        "text": "Unexpected single-line block (block-no-single-line)",
      },
    ],
  },
]
`;
